src/GitVersionExe/HelpWriter.cs

```diff
@@ -24,8 +24,8 @@ public static void WriteTo(Action<string> writeAction)
     /showvariable   Used in conjuntion with /output json, will output just a particular variable. 
                     eg /output json /showvariable SemVer - will output `1.2.3+beta.4`
     /l              Path to logfile.
-    /showconfig     Outputs the effective GitVersion config (defaults + custom from GitVersion.yaml) in yaml format
-    /overrideconfig Overrides GitVersion config values inline (semicolon-separated key value pairs e.g. /overrideconfig:tag-prefix=Foo)
+    /showconfig     Outputs the effective GitVersion config (defaults + custom from GitVersion.yml) in yaml format
+    /overrideconfig Overrides GitVersion config values inline (semicolon-separated key value pairs e.g. /overrideconfig tag-prefix=Foo)
```

Comment:
Added note about GitConfig.yml 

I think It worth add section about override config and show config in separate issue (the same I proposed on parsing overriden parameters.